relude.cabal

```diff
@@ -15,7 +15,7 @@ description:
       you can still use some unsafe functions from @Relude.Unsafe@ module, but they
       are not exported by default.
     .
-    * __Type-safety__. We like to make invalid states unrepresantable. And if it's
+    * __Type-safety__. We like to make invalid states unpresentable. And if it's
```

Comment:
I'm not a native speaker, but I want the meaning to be _not possible to represent invalid states_ thus using the word `unrepresentable`. _Unpresentable_ is

> not clean, well-dressed, or decent enough to be seen in public

which seems to have a different meaning to me.

Reply:
Whoops, yes, in correcting the typo, I introduced a different typo. I only meant to change an "e" to an "a".